buildSrc/src/main/kotlin/com/bmuschko/gradle/docker/DocumentationPlugin.kt

```diff
@@ -0,0 +1,59 @@
+package com.bmuschko.gradle.docker
+
+import org.ajoberstar.gradle.git.publish.GitPublishExtension
+import org.ajoberstar.gradle.git.publish.GitPublishPlugin
+import org.asciidoctor.gradle.AsciidoctorTask
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.javadoc.Groovydoc
+import org.gradle.api.tasks.javadoc.Javadoc
+import org.gradle.external.javadoc.StandardJavadocDocletOptions
+import org.gradle.kotlin.dsl.*
+
+class DocumentationPlugin : Plugin<Project> {
+    override fun apply(project: Project): Unit = project.run {
+        applyGitPublishPlugin()
+        addLinks()
+        configureGitPublishExtension()
+    }
+
+    private
+    fun Project.applyGitPublishPlugin() {
+        plugins.apply(GitPublishPlugin::class.java)
```

Comment:
Can be simplified to `apply<GitPublishPlugin>()`

Reply:
done